otx/algorithms/classification/utils/label.py

```diff
@@ -0,0 +1,94 @@
+"""Collection of utils about labels in Classifation Task."""
+
+# Copyright (C) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+# pylint: disable=too-many-nested-blocks
+
+import importlib
+from operator import itemgetter
+from typing import List
+
+from otx.api.entities.label import Domain, LabelEntity
+from otx.api.entities.label_schema import LabelGroup, LabelGroupType, LabelSchemaEntity
+from otx.api.utils.argument_checks import check_input_parameters_type
+
+
+@check_input_parameters_type()
+def generate_label_schema(not_empty_labels: List[LabelEntity], multilabel: bool = False):
```

Comment:
I forgot what the purpose of `generate_label_schema` was. It looks like we need to take a look at this together.
Even, `otx.cli.utils.io` has a function with the same name.

Reply:
I named it as label.py because all functions are related to labels. and this generate label schema is different from detection utils cause it has multilabel arguments. If all agree that naming the file to utils.py is appropriate, I will change.

Reply:
I revised folder name to utils. However, this generate_label_schema function is different from detection, so left it as is.